.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Ready for review checklist:
+- [ ] A self-review has been performed
+- [ ] All commits run individually
+- [ ] All TODO, console.log or other temporary changes have been removed
+- [ ] The PR has been tested locally
+- [ ] A test have been written
+  - [ ] This change doesn't need a new test
+- [ ] Relevant issues are linked
+- [ ] Remaining work is documented in issues
+  - [ ] There is no remaining work from this PR that require new issues
```

Comment:
Should we add
`- [ ] All pipelines run successfully`
Or perhaps we cannot check this off when creating PR? (depends on if this list is used on PR create or PR ready for review)

Reply:
No we need this when we also have the pipelines further down that gives the same check. Should be included in private repos though

Reply:
![image](https://github.com/user-attachments/assets/e5ede817-5413-46f9-8b4d-339c01ca5c8c)
The checklist will appear in the marked area and cannot be checked until after the pr is created

Reply:
Then I agreed we dont add this =)